README.md

```diff
@@ -20,6 +20,7 @@ Set the supported components to monitor and the tunings like number of iteration
 cerberus:
     kubeconfig_path: ~/.kube/config                      # Path to kubeconfig
     watch_nodes: True                                    # Set to True for the cerberus to monitor the cluster nodes
+    watch_operators: True                                # Set to True for cerberus to monitor cluster operators. Parameter is optional, will set to false if not specified
```

Comment:
Let's enable this by default in the config ( https://github.com/openshift-scale/cerberus/blob/master/config/config.yaml) and add it to the https://github.com/openshift-scale/cerberus#kubernetesopenshift-components-supported section of the readme as monitoring cluster operators is critical to determining the stability of the cluster. Thoughts?

Reply:
Added this key to the config.yaml with value True. Setting the default value to true if the key is not set. Adding it to the bottom section of the readme.